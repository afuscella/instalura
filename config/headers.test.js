/* eslint-disable no-console */
import headers from './headers';

console.log(headers);
describe('config/headers', () => {
  test('when headers is requested should match the static def', () => {
    const headersExpected = headers;
    const headersActual = [
      {
        source: '/app/:path*/',
        headers: [
          {
            key: 'X-Frame-Options',
            value: 'DENY',
          },
        ],
      },
    ];
    expect(headersExpected).toEqual(headersActual);
  });

  test('when headers is requested should match the snapshot', () => {
    const headersExpected = headers;
    expect(headersExpected).toMatchSnapshot();
  });
});
