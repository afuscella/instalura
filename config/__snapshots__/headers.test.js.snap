// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config/headers when headers is requested should match the snapshot 1`] = `
Array [
  Object {
    "headers": Array [
      Object {
        "key": "X-Frame-Options",
        "value": "DENY",
      },
    ],
    "source": "/app/:path*/",
  },
]
`;
